shared/js/background/settings.es6.js

```diff
@@ -31,12 +31,45 @@ function ready () {
     return _ready
 }
 
+// Ensures we have cleared up old storage keys we have renamed
```

Comment:
👏 Nice! I remember Brad also added clean up function somewhere - should those be consolidated?

Reply:
@kdzwinel I added "cookieExcludeList" to this list as it should have been removed. I created a follow up https://app.asana.com/0/892838074342800/1200766487104196/f task to clear up other storage usages and document it.